css/index.css

```diff
@@ -167,31 +173,32 @@ nav .logo {
 }
 
 header {
-	line-height: 0.75;
-	margin: 50px 0px 50px 100px;
-	width: 876px;
+	margin: 50px auto;
```

Comment:
don't need to use margin auto on the left and right here. flexbox should be what's used to center header element